CONTRIBUTING.md

```diff
@@ -28,7 +28,8 @@ We aim at creating a new release everyone 4 to 6 weeks.
 The decision on creating a new release is made during the bi-weekly calls.
 A release consists of of merging `develop` into `stable` and create a new tag for that version
 using a modified [semantic versioning](https://semver.org/) scheme.
-Releases will be tagged `0.MAJOR.MINOR` given the current rapid development.
+Releases will be tagged `v0.MAJOR.MINOR` given the current rapid development.
+Release branches named `v0.MAJOR` will track the latest minor release.
```

Comment:
Would this result in having a persistent release branch for every major release?
I'm wondering what the use case is compared to directly using the tags, e.g., if I want to update the Parthenon submodule in a downstream code I'd have to pull the changes and commit the updated version to the downstream repo anyway.

Reply:
Yeah, I suppose. I think it just makes it easier to get the latest ABI compatible version if you want to update from an older 0.1 to a newer 0.1, without having to know what the latest minor release is.

Reply:
Given the current rapid change I suggest that we hold off the separate branch until the first official 1.x release.
For now we could add a hint to the docs saying that with a simple `git tag --list 'v0.MAJOR.*'` one can find the latest minor release.